CHANGELOG.md

```diff
@@ -25,6 +25,26 @@ The lerna-changelog tool detects changes based on PR labels and maps them to sec
 -->
 
 
+## [v1.12.0] (2021-03-24)
+
+#### :rocket: Added
+* [#1841](https://github.com/SAP/luigi/pull/1841) Bookmarkable modals ([@maxmarkus](https://github.com/maxmarkus))
+* [#1935](https://github.com/SAP/luigi/pull/1935) Add warning if both virtualTree and children nodes are defined  ([@ndricimrr](https://github.com/ndricimrr))
+* [#1894](https://github.com/SAP/luigi/pull/1894) Add more flexibility to control handshake ([@legteodav](https://github.com/legteodav))
+* [#1897](https://github.com/SAP/luigi/pull/1897) Update fiddle to v1.11.0 ([@JohannesDoberer](https://github.com/JohannesDoberer))
+
+#### :bug: Fixed
+* [#1945](https://github.com/SAP/luigi/pull/1945) Quick Fix Console Warning ([@ndricimrr](https://github.com/ndricimrr))
```

Comment:
```suggestion
```
I'd remove this, as its follow-up of 1935 in Added section

Reply:
Aside from that, looks good 👍

Reply:
thanks.. updated in the description.. but didn't commit it :)